### DISCUSSION TITLE
Question: splitting 'slaves' horizontally ?

### DISCUSSION THREAD

**Discussion by fZ0x539** [hyprwm/Hyprland#1737] [Q&A]
Working with my code editor I'd prefer to have it taking up half the screen, then the browser the other half, but below the browser I'd like 2-3 terminals (ultrawide monitor) stacked like a pyramid
so:

Master
mini-Master
Slave

Edit: here's an exact photo of what I want, only that it's using dwindle and I'd prefer to have it in master layout
https://imgur.com/iGV2y0B

I've set the orientationright but I have absolutely no idea what this kind of layout is called nor what dispatcher I'm looking for, but I'm guessing it's related to groups? Included two terminals in a group but wasn't able to place them horizontally. 
I know that the Dwindle layout could do this, but I'm wondering whether it's possible using a master layout?

**Comment by vaxerski** [comment]
I'm not sure I understand but this looks like `orientationtop`?

**Reply by fZ0x539** [reply]
> I'm not sure I understand but this looks like `orientationtop`?

Nah man :/ `orientationtop` gives me a full width window on top https://imgur.com/e6hvVhT

Lemme find a photo https://imgur.com/iGV2y0B
This is using dwindle, which I have no idea how to setup or work with. I'm wondering whether this is doable using Master Layout?

**Reply by vaxerski** [reply]
master cannot do that. Just switch to dwindle?

**Reply by fZ0x539** [reply]
> master cannot do that. Just switch to dwindle?

Oh I see, well thank you for clarifying!
